Javascript/research.js

```diff
@@ -284,7 +284,7 @@ resdesc = [null,
     "[7x9] Somehow, I can't explain why, you reduce your taxes by 2% multiplicative, based on 3/5 * log10(Rare Fragments)!",
     "[7x10] Want a permanent blessing boost? I know you do. A permanent +25% effect to all blessings.",
     "[7x11] +0.3% Rune Effectiveness / level [NOTE TO TESTER: I did not really have time to make flavor texts after 7x10 sorry!]",
-    "[7x12] +0.01% to Inceptus power / level",
+    "[7x12] +0.0001% to Inceptus power / level",
```

Comment:
Referring to https://github.com/Pseudonian/SynergismOfficial/blob/501f74b875c7641464e56b60ec5ebea41ac910c6/Synergism.js#L1901.